contracts/dapis/DapiServer.sol

```diff
@@ -710,6 +699,32 @@ contract DapiServer is
         return dataFeed.value;
     }
 
+    /// @notice Aggregates the Beacons and returns the result
+    /// @dev Tha aggregation of Beacons may have a different value than the
+    /// respective Beacon set, e.g., because the Beacon set has been updated
+    /// using signed data
+    /// @param beaconIds Beacon IDs
+    /// @return value Aggregation value
+    /// @return timestamp Aggregation timestamp
+    function aggregateBeacons(bytes32[] memory beaconIds)
+        public
+        view
+        override
+        returns (int224 value, uint32 timestamp)
+    {
+        uint256 beaconCount = beaconIds.length;
+        require(beaconCount > 1, "Specified less than two Beacons");
+        int256[] memory values = new int256[](beaconCount);
+        uint256 accumulatedTimestamp = 0;
+        for (uint256 ind = 0; ind < beaconCount; ind++) {
+            DataFeed storage dataFeed = dataFeeds[beaconIds[ind]];
```

Comment:
This variable seems to me like it could just use the `memory` modifier since we are not going to change the data on the underlying mapping but just use it to read it and store it temporarily. This is a `view` function after all 😄

Reply:
`storage` is the cheaper one, if you use `memory` it gets copied from storage to memory, which will induce the gas overhead